spec/models/archived_petition_spec.rb

```diff
@@ -25,7 +21,7 @@
     end
 
     it "sorts the results by the created_at timestamp" do
-      expect(ArchivedPetition.search(q: "Petition")).to eq([petition_2, petition_1])
+      expect(ArchivedPetition.search(q: "Petition")).to match_array([petition_2, petition_1])
```

Comment:
Isn't the order of the elements in the array important, why change to `match_array`?